dockerfiles/keycloak/che/login/messages/messages_en.properties

```diff
@@ -6,3 +6,6 @@ formRegister=Create an account with a form
 passwordRecover=Forgot your password? <br/> Enter your email below.
 
 usernameTitle=Username has to comply with the DNS naming convention. An alphanumeric (a-z, and 0-9) string, with a maximum length of 63 characters, with the '-' character allowed anywhere except the first or last character.
+
```

Comment:
do we need new lines between properties?

Reply:
no, removed.